auth/src/main/java/com/farsitel/bazaar/storage/connection/StorageConnection.kt

```diff
@@ -14,21 +18,69 @@ internal abstract class StorageConnection(private val context: Context) {
     abstract fun saveData(owner: LifecycleOwner?, data: ByteArray, callback: BazaarStorageCallback)
     abstract fun saveDataSync(owner: LifecycleOwner?, data: ByteArray)
 
+    open fun disconnect(context: Context) {
+        storageConnection = null
+    }
+
+    fun getGetSavedDataResponse(extras: Bundle?): BazaarResponse<ByteArray> {
```

Comment:
Two get in the name is a little confusing, You can use something like `retriveGetSavedDataResponse`

Reply:
Done